tilings/strategies/verification.py

```diff
@@ -184,9 +185,12 @@ def pack(tiling: Tiling) -> StrategyPack:
         )
 
     def verified(self, tiling: Tiling) -> bool:
-        return (
-            tiling.dimensions == (1, 1)
-            and frozenset(ob.patt for ob in tiling.obstructions) not in self.symmetries
+        return tiling.dimensions == (1, 1) and (
```

Comment:
I'm kind of surprised that we verify the root if it has an assumption on it. I feels circular

Reply:
See my initial comment on the PR.